tests/armature/Node.spec.ts

```diff
@@ -81,15 +205,21 @@ describe('Node', () => {
                 .pointAt(vec3.fromValues(0, 0, -2))
                 .release();
 
+            expect(parent.getRotation()).toEqualMat4(
+                mat4.fromQuat(mat4.create(), quat.fromEuler(quat.create(), -90, 0, 0))
+            );
+
             child
                 .grab(child.point('tip'))
                 .pointAt(vec3.fromValues(0, 1, -1))
                 .release();
 
-            expect(child.getRotation()).toEqualQuat(quat.fromEuler(quat.create(), 90, 0, 0));
+            expect(child.getRotation()).toEqualMat4(
+                mat4.fromQuat(mat4.create(), quat.fromEuler(quat.create(), 90, 0, 0))
+            );
         });
 
-        it('can rotate constrained to an axis a node to look at a point in another node', () => {
+        fit('can rotate constrained to an axis a node to look at a point in another node', () => {
```

Comment:
Why is this `fit` and not `it`?

Reply:
woops debug code I forgot to take this out while debugging